src/dbg/dbg.c

```diff
@@ -130,7 +130,7 @@ void dbg_handler_set(dbg_print_h *ph, void *arg)
 
 
 /* NOTE: This function should not allocate memory */
-static void dbg_vprintf(int level, const char *fmt, va_list ap)
+static void dbg_vprintf(struct pl *id, int level, const char *fmt, va_list ap)
```

Comment:
`const` for id ?

Reply:
I think a custom log handler should able to call `mem_ref(id)`, this discards the const qualifier.